optimism.tokenlist.json

```diff
@@ -773,6 +773,50 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
+    }, 
+    {
+      "chainId": 1,
+      "address": "0x0391D2021f89DC339F60Fff84546EA23E337750f",
+      "name": "BarnBridge Governance Token",
+      "symbol": "BOND",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BOND.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0x3e7eF8f50246f725885102E8238CBba33F276747",
+      "name": "BarnBridge Governance Token (Optimism)",
```

Comment:
Can you make sure the name here is the same across all networks. Basically change it to "BarnBridge Governance Token"

Reply:
i can, but just to make sure, the contract itself is depolyed with the name BarnBridge Governance Token (Optimism)
is it ok to have it different just in this json?
cheers

Reply:
Just merged #55 for kovan and successfully tested there. 

Deposit https://kovan.etherscan.io/tx/0x1338ab4e57a498c50c4b6577da0410df7f802a1e371f639124e19adb4d0958d6
https://kovan-optimistic.etherscan.io/tx/0xe8ad19bad33deddcba43addfa60bf6b4289c96d00e0b3d2879cdf0611b083087

Withdrawal
https://kovan-optimistic.etherscan.io/tx/0x6eb468ca4a24e8ba7ad68c3af6f0f5a3a462a1bdd99b8a3eb7c8771a30766a45
https://kovan.etherscan.io/tx/0x1d259b57892d88e4ea3ffdacbbd9eb5aba2567c0a9ad6e449d3c61b55df45538